### DISCUSSION TITLE
Question 79: The issue of fathoms (IZ)

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#868] [Q&A] [answered]
Description: There is no field for fathoms in the site module Elevation (relative). Many of our dredge depths are in fathoms, and conversion results in some value drift/rounding. If possible, we suggest adding depths in the original unit entered in EMu (in black). Otherwise, using the number entered in the depth/feet field of the collection event module, as the unit feet causes the least unit conversion drift from fathoms and meters.

ZD 63529

**Comment by rondlg** [comment, marked as answer]
Response: Zendesk 63529 - See github: https://github.com/fieldmuseum/EMu-Documentation/discussions/829

INTERIM workflow: use ft fields because the conversion is better than from metres.

Fathoms in sites is a development request for v9 which has more screen real estate to fit it in. Records also have inches recorded so will need to account for that.

**Comment by fmrdelapena** [comment]
Can we please add inches to the proposed use of the Relative Elevation table in the Site Module.  This would allow for inches, feet, meters and fathoms columns.
